collectors/statsd.plugin/README.md

```diff
@@ -19,7 +17,7 @@ Since statsd is embedded in Netdata, it means you now have a statsd server embed
 
 Netdata statsd is fast. It can collect more than **1.200.000 metrics per second** on modern hardware, more than **200Mbps of sustained statsd traffic**, using 1 CPU core. The implementation uses two threads: one thread collects metrics, another one updates the charts from the collected data.
 
-# Available StatsD collectors
+## Available StatsD collectors
```

Comment:
Since you are removing H1 (`# statsd.plugin`), should we make this one and other H2 headers H1?

Reply:
Ok, I see we [can't have multiple H1](https://github.com/netdata/netdata/pull/11764/files#r780462291).